### DISCUSSION TITLE
Setup / run / dev question (uart etc)

### DISCUSSION THREAD

**Discussion by ixmeins** [picocomputer/community#10] [Q&A] [answered]
My board lives. I've setup a new VM for dev stuff and have a working vscode instance and I've modifed rp6502.py for my serial port and baudrate.
For general dev should my USB serial bridge be connected to the RIA UTX/URX pins and the uart pins on the VGA left disconnected so I can upload and use a remote serial session or am I missing something?
When I compile and run the brot32.c example app I don't get graphics on the VGA but XREG: $2FF $263A (as an example) scrolling up the screen. I'm clearly missing something!
Cheers,
Ingmar

**Comment by ixmeins** [comment]
Okay, I worked out that I should use the /ttyACM0 connection provided by the VGA module and have the UTX/URX lines connected between the RIA and VGA. Doh... 
Not solved the XREG: messages instead of graphics yet.

**Reply by rumbledethumps** [reply, marked as answer]
Check your PIX bus. Looks like PIX1 is stuck on.
